docs/tutorials/ruby-driver-authentication.txt

```diff
@@ -179,6 +180,166 @@ see the :manual:`X.509 tutorial in the MongoDB Manual
                              ssl_ca_cert: '/path/to/ca.pem' )
 
 
+AWS
+```
+
+*Requires MongoDB Enterprise Edition and server version 4.4 or later.*
```

Comment:
Will this be available in Atlas? If so, I'd suggest adding "Requires MongoDB Atlas or ...".

Reply:
We do mention Atlas in a couple of places in documentation but we do not call it out as supported enterprise features. Do you find that users perceive a lack of clarity as to what functionality can be used with Atlas?